src/reaper-vault-erc4626.ts

```diff
@@ -59,79 +65,73 @@ export function handleStrategyReported(event: StrategyReported): void {
     losses.toString(),
     allocated.toString(),
     allocationAdded.toString(),
-    allocBPS.toString()
+    allocBPS.toString(),
   ]);
 
-  const strategy = Strategy.load(strategyId);
-  if (strategy !== null) {
-    let currentReportId = strategy.latestReport;
-    log.info(
-      '[Strategy] Getting current report {} for strategy {}.',
-      [currentReportId ? currentReportId : 'null', strategy.id]
-    );
-    let strategyReportId = buildIdFromEvent(event);
-    let strategyReport = StrategyReport.load(strategyReportId);
-    if (strategyReport === null) {
```

Comment:
also I'm not sure why there was a null check here, `strategyReported` _must_ be null since we just built a new ID for it from the event. null check is removed below.